frontend/src/pages/ActivityReport/Pages/components/RecipientReviewSection.js

```diff
@@ -56,8 +60,50 @@ const RecipientReviewSection = () => {
     },
   ];
 
-  const buildGoalReview = (goal) => goalSection[0].items.map((item) => (
-    <>
+  const buildFeiRootCauseReviewSection = (item, goal) => {
+    const promptsForReview = goal.promptsForReview || [];
+    return (promptsForReview.length > 0 && (
+      <div className="grid-row margin-bottom-3 desktop:margin-bottom-0 margin-top-1">
+        {promptsForReview.map((v, index) => (
+          <>
+            <div className="grid-col-12 desktop:grid-col-6 print:grid-col-6  font-sans-2xs desktop:font-sans-sm text-bold desktop:text-normal">
+              {item.label}
+            </div>
+            <div className="grid-col-12 desktop:grid-col-6 print:grid-col-6">
+              <div aria-label={`${item.label} ${index + 1}`} key={`${item.label}${v}`} className="desktop:flex-align-end display-flex flex-column flex-justify-center">
```

Comment:
please remove the ARIA label here, it will override the contents

Reply:
done